bitbakery/bitbake.sh

```diff
@@ -12,17 +12,12 @@ export HOME=/home/bitbake
 source $SRC_DIR/meta-updater/scripts/envsetup.sh $TARGET $BUILD_DIR
 
 # appending user-defined config
-cp /opt/site.conf conf/
+cp $CONF_DIR/site.conf conf/
 
 # build image
 bitbake $BITBAKE_FLAGS $IMAGE
-result=$?
 
-if [ $result -eq 0 ]; then
-  cp $BUILD_DIR/tmp/deploy/images/$TARGET/$IMAGE-$TARGET.otaimg $OUT_DIR
-  cp $BUILD_DIR/tmp/deploy/images/$TARGET/u-boot.* $OUT_DIR
-  cp -r $BUILD_DIR/tmp/deploy/images/$TARGET/ostree_repo $OUT_DIR
-fi
-
-exit $result
+cp $BUILD_DIR/tmp/deploy/images/$TARGET/$IMAGE-$TARGET.otaimg $OUT_DIR
```

Comment:
If we can't check exit code and branch, can we perhaps run it with -x to make the script shortcut when a command fails (so it doesn't reach the `cp`s)?